src/example_node_wallet.ts

```diff
@@ -0,0 +1,27 @@
+import { Keypair, TransactionBuilder, hash } from '.'
+
+/**
+ * For use with {@link ContractClient} and {@link AssembledTransaction}.
+ * Implements `signTransaction` and `signAuthEntry` with signatures expected by
+ * those classes. This is useful for testing and maybe some simple Node
+ * applications. Feel free to use this as a starting point for your own
+ * Wallet/TransactionSigner implementation.
+ */
+export class ExampleNodeWallet {
```

Comment:
suggestion: `BasicNodeSigner` to both remove the "example" implication and "wallet" nomenclature